doc_source/resource-type-cli-invoke.md

```diff
@@ -30,6 +31,10 @@ The logical Lambda function name in the SAM template\. Alternately, you can also
 
 Default: `TypeFunction`
 
+`--profile <value>`
+
+The AWS profile to use\. If no profile is specified, the client applies credentials specified in the [Boto3 credentials chain](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/configuration.html)\.
+
```

Comment:
Would be useful to show how profile should be specified

Reply:
Would you mind elaborating on this, please? We have the `--profile <value>` example above on line 34, consistent with other documented options.